hrui/js/src/hrui.js

```diff
@@ -271,14 +273,15 @@
   }
 
   /**
-   * Moves the "Inline Custom Data" block towards the top of the
-   * personal details tab
+   * Moves the specified selector element to the personal details tab
+   *
+   * @param selector
    */
-  function repositionInlineCustomDataBlockInPersonalDetailsTab () {
-    $('.Inline_Custom_Data')
+  function repositionPersonalDetailsBlock (selector) {
+    $(selector)
       .removeClass('crm-collapsible collapsed')
       .addClass('crm-summary-block')
-      .insertAfter('.crm-summary-contactinfo-block')
+      .appendTo('.contactCardLeft')
```

Comment:
please check the DOM you are manipulating. There are multiple `.contactCardLeft` in the page, that's why I wrote `.contactTopBar .contactCardLeft` in my comment

Reply:
Oh i missed that comment. :(